script/hassfest/manifest.py

```diff
@@ -113,6 +113,27 @@ class QualityScale(IntEnum):
     "websocket_api",
     "zone",
 ]
+# Grandfather rule for older integrations
+# https://github.com/home-assistant/developers.home-assistant/pull/1512
+NO_DIAGNOSTICS = [
```

Comment:
We should create issues for these integrations to notify the code owner.

Reply:
Done in https://github.com/home-assistant/core/issues/117576